docs/examples/cas9_kymotracking/cas9_kymotracking.rst

```diff
@@ -0,0 +1,231 @@
+brackets while hovering over the kymograph). The `line width` should roughly be set to the expected line width of a
+line. The `window` should be chosen such that small gaps in a trace can be overcome, but not so large that spurious
+points may be strung together as a trace. `Sigma` controls how much the location can fluctuate from one point to the
+next, while the `min length` determines how many peak points should be in a trace for it to be considered a valid
+trace.
+
+Holding down the left mouse button and dragging pans the view, while the outer mouse button allows us to drag a region
+where we should trace lines. Any line which overlaps with the selected area will be removed before tracing new ones.
+
+The icon with the little square can be used to toggle zoom mode, which will allow you to zoom in one subsection of the
+kymograph. Clicking it again brings us back out of zoom mode. You can zoom out again by clicking the home button. Quite
+often, it is beneficial to find some adequate settings for track all, and then fine tune the results using the manual
+rectangle selection. It’s not mandatory to use the same settings throughout the kymograph. For example, if you see a
+particular event where two lines are disconnected that should be connected, temporarily increase the window size and
+just drag a rectangle over that particular line with `Add lines` enabled.
+
+Now, let’s track some traces. There are two ways to approach this analysis. The first is to just use the rectangle
+selection, which can be quite time intensive. Alternatively, you can use `Track all` to simply track all lines found
+in the kymograph, and then remove spurious detections by hand. This can be good to get a feel for the parameters as
+well. If we then disable the toggle `Add lines` we will start removing lines without grabbing new ones. This
+functionality can be used to remove spurious detections.
+
+Note that in this data for example, there are some regions where fluorescence starts building up on the surface of the
+bead. This binding should be omitted from the analysis::
+
+    kymowidget = lk.KymoWidgetGreedy(data, axis_aspect_ratio=2, min_length=4, pixel_threshold=3, window=6, sigma=1.4, vmax=8)
+
+.. image:: kymowidget.png
+
+One last thing to note is that we assigned the `KymoWidgetGreedy` to the variable `kymowidget`. That means that from
+this point on, we can interact with it through the handle name `kymowidget`.
+
+Exporting from the widget results in a file that contains the line coordinates in pixels. If we wish to calibrate them,
+we can export manually and pass a `dt` and `dx` argument which correspond to the pixel size. If we also want to
+export the photon counts in a region around the traced line, we can include a `sampling_width`. This sums the photon
+counts from `pixel_position - sampling_width` to (and including) `pixel_position + sampling_width`::
+
+    kymowidget.save_lines("kymotracks_calibrated.txt", dt=dt, dx=pixel_size, sampling_width=3)
```

Comment:
Shouldn't the second `dt` be something else here....like a differently named variable from some previous PR?

Reply:
I'm not sure what you mean here?

The variable was called `dt` above (`dt = downsample_factor * kymo.line_time_seconds`) and the second argument of save is `dt`:
```python
def save_lines(self, filename, dt=None, dx=None, delimiter=';', sampling_width=None):
```

Reply:
Ah, I see. I was referring to a previous PR where we decided on distinguishing parameters from local variables. Doesn't apply here it seems.